app/src/lib/stores/app-store.ts

```diff
@@ -206,6 +210,10 @@ const FastForwardBranchesThreshold = 20
 
 const LastSelectedRepositoryIDKey = 'last-selected-repository-id'
 
+const RecentRepositoriesKey = 'recently-selected-repositories'
+export const RecentRepositoriesLength = 5
```

Comment:
also, is there a better place for this constant to live? i couldn't find anywhere that made more sense in our current organization, but maybe i missed something?

Reply:
I think here is fine (see `const FastForwardBranchesThreshold = 20` as another example), because I also think you can drop the `export` from this line (not sure why I didn't notice that last round.